jbmc/src/java_bytecode/java_bytecode_language.h

```diff
@@ -201,13 +225,18 @@ class java_bytecode_languaget:public languaget
     const irep_idt &function_id,
     symbol_table_baset &symbol_table)
   {
+    lazy_class_to_declared_symbols_mapt class_to_declared_symbols{};
```

Comment:
:question: Why does this method create its own `lazy_class_to_decared_symbols_mapt`? Could this cause any problems or inconsistencies?

Reply:
No inconsistency because if it is redeclared everytime this preserves the old behaviour. But yes it's not optimal I will make sure it is declared by the function which calls that one on each methods.